requirements-dev.txt

```diff
@@ -44,7 +44,7 @@ google-cloud-core==1.0.3
 google-cloud-storage==1.19.0
 google-resumable-media==0.3.3
 googleapis-common-protos==1.6.0
-httpie==1.0.2
+httpie==1.0.3
```

Comment:
This file should not be edited directly.

Please edit `requirements-dev.txt.in` instead, and update `requirements-dev.txt` with `make requirements-dev.txt`.